components/automate-ui/src/app/pages/data-feed/data-feed.component.html

```diff
@@ -26,11 +27,18 @@
       </chef-page-header>
       <div class="page-body">
         <div *ngIf="(sortedDestinations$ | async)?.length === 0" class="empty-state">
-          <p>Create a data feed to get started!</p>
+          <img alt="No preview" src="/assets/img/no_preview.gif" />
+          <p>No Data Feeds Preview Available</p>
         </div>
         <div [ngClass]="(sortedDestinations$ | async)?.length === 0 ? 'empty-state' : ''">
           <!-- <chef-button primary [routerLink]="['/settings/data-feed/form']">Create Data Feed</chef-button> -->
-          <chef-button primary (click)="openCreateModal()">Create Data Feed</chef-button>
+          <chef-button primary (click)="openCreateModal()">
+            <svg aria-hidden="true" focusable="false" data-prefix="fal" data-icon="plus-circle" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" 
```

Comment:
for this you can do like

```
.dex-btn-icon--saml {
  background-color: #84B6EF; 
  background-image: url(../static/img/saml-icon.svg);
}
```
find this saml-icon.svg and check how we can do using svg file.

Reply:
Added svg file